cthreads.c

```diff
@@ -2,6 +2,11 @@
 #include <stdlib.h>
 #include <stdint.h>
 
+#include <errno.h>  /* For errno */
```

Comment:
```suggestion
#include <errno.h>  /* errno */
```

Reply:
Oh wait, I need to put that into a `#ifdef` anyway.